doc/manual/manual.tex

```diff
@@ -6310,11 +6310,78 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 parameter (the division by 0.02 in the formula for the smoothed initial
 condition). However, for more exact results, one can choose
 this half width parameter proportional to the mesh size and thereby get more
-accurate results. Since the point of the section was to demonstrate the reason
-for the lack of convergence, but we leave this as an experiment to the reader.
+accurate results. The point of the section was to demonstrate the reason
+for the lack of convergence.
 
+In this section we extend the Van Keken cookbook following up the work previously completed by Cedric Thieulot, Juliane Dannberg,
+Timo Heister and Wolfgang Bangerth.  \textit{This section contributed by Grant Euen, Tahiry Rajaonarison, and Shangxin Liu as part of the Geodynamics and ASPECT class at Virginia Tech.}  
 
+As already mentioned above, using a half width parameter proportional to the mesh size allows for more accurate results.  We test the effect of the half width size of the smoothed discontinuity by changing the division by 0.02, the smoothing parameter, in the formula for the smoothed initial conditions into values proportional to the mesh size.  We use global refinement degree 7 because the root mean square velocity converges at greater refinement degree while keeping average runtime around 5 to 25 minutes.  These runtimes were produced by the BlueRidge cluster of the Advanced Research Computing (ARC) program at Virginia Tech.  BlueRidge is a 408-node Cray CS-300 cluster; each node outfitted with two octa-core Intel Sandy Bridge CPUs and 64 GB of memory.  A chart of average runtimes for global mesh refinements 5 through 10 on one node can be seen in Table~\ref{tab:runtime-table}.  For global refinement degree 7 (128$\times$128 mesh size), the size of the mesh is 0.0078 corresponding to a half width parameter of 0.0039.  The smooth model allows for much better convergence of the secondary plumes, although they are still more scattered than the primary plumes.
 
+\begin{table}[htb]
+	\center
+	\begin{tabular}{|c|cccccc|}
+		\hline
+		Global & & Number of Processors & & \\
+		Refinements & 4 & 8 & 12 & 16
+		\\ \hline
+		5 & 28.1 seconds & 19.8 seconds & 19.6 seconds & 17.1 seconds \\
+		6 & 3.07 minutes & 1.95 minutes & 1.49 minutes & 1.21 minutes \\
+		7 & 23.33 minutes & 13.92 minutes & 9.87 minutes & 7.33 minutes \\
+		8 & 3.08 hours & 1.83 hours & 1.30 hours & 56.33 minutes \\
+		9 & 1.03 days & 15.39 hours & 10.44 hours & 7.53 hours \\
+		10 & More than 6 days & More than 6 days & 3.39 days & 2.56 days \\ \hline
+	\end{tabular}
+	\caption{\it Average runtimes for the Van Keken Benchmark with smoothed initial conditions.  All of these tests used different numbers of processors on one node on the BlueRidge cluster of ARC at Virginia Tech.}
+	\label{tab:runtime-table}
```

Comment:
Alright, I updated the table's caption with that information and pushed the
manual.tex file again.  Does everything look good?

On Tue, Dec 8, 2015 at 3:36 PM, Wolfgang Bangerth <notifications@github.com>
wrote:

> In doc/manual/manual.tex
> <https://github.com/geodynamics/aspect/pull/683#discussion_r47009874>:
>
> > +\begin{table}[htb]
> > +	\center
> > +	\begin{tabular}{|c|cccccc|}
> > +		\hline
> > +		Global & & Number of Processors & & \\
> > +		Refinements & 4 & 8 & 12 & 16
> > +		\\ \hline
> > +		5 & 28.1 seconds & 19.8 seconds & 19.6 seconds & 17.1 seconds \\
> > +		6 & 3.07 minutes & 1.95 minutes & 1.49 minutes & 1.21 minutes \\
> > +		7 & 23.33 minutes & 13.92 minutes & 9.87 minutes & 7.33 minutes \\
> > +		8 & 3.08 hours & 1.83 hours & 1.30 hours & 56.33 minutes \\
> > +		9 & 1.03 days & 15.39 hours & 10.44 hours & 7.53 hours \\
> > +		10 & More than 6 days & More than 6 days & 3.39 days & 2.56 days \\ \hline
> > +	\end{tabular}
> > +	\caption{\it Average runtimes for the Van Keken Benchmark with smoothed initial conditions.  All of these tests used different numbers of processors on one node on the BlueRidge cluster of ARC at Virginia Tech.}
> > +	\label{tab:runtime-table}
>
> Would you mind updating the pull request with this information? (Just add
> another commit and push it to your github account.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/aspect/pull/683/files#r47009874>.
>



-- 
Grant Euen
Graduate Student
Department of Geosciences
Virginia Tech
Blacksburg, VA 24061